wielder/wield/remote.py

```diff
@@ -88,12 +88,18 @@ def get_remote_unique_context(conf):
     conf_path = f'{bucket_path}/{namespace_bucket}/{unique_config_path}'
     file_name = f'{unique_name}.conf'
 
-    if conf.runtime_env == 'aws':
-        b = AWSBucketeer(conf)
+    if conf.runtime_env not in local_deployments:
+        runtime_env = RuntimeEnv[conf.bootstrap_env.upper()]
+        bucket_env = RuntimeEnv[conf.runtime_env.upper()]
 
+        b = get_bucketeer(conf, runtime_env, bucket_env)
         b.download_object(namespace_bucket, key=unique_config_path, name=file_name, dest=conf_path)
 
-    conf = wu.parse_file_as_hocon(f'{conf_path}/{file_name}')
```

Comment:
sometimes a non existent file should throw an exception. Do you have a reason for changing this pattern?